src/Symfony/Component/HttpKernel/Attribute/HttpException/AccessDenied.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute\HttpException;
```

Comment:
```suggestion
namespace Symfony\Component\HttpKernel\Attribute;
```

I think this should be moved up a level (but keep the extended attributes in the sub-namespace).

Reply:
I thought it's better to group these new attributes in a separate namespace as the list is a bit long and the other attributes (such as `\Symfony\Component\HttpKernel\Attribute\AsController`) will get lost between them.

Reply:
Yes, I was thinking just `HttpException` to be moved up a level. So when using it on its own you don't have the duplicated name in your use statement (this is just a minor eyesore for me 😉).

```php
use Symfony\Component\HttpKernel\Attribute\HttpException\HttpException;

// vs

use Symfony\Component\HttpKernel\Attribute\HttpException;
```

Reply:
That makes sense. I misunderstood that you meant all the attributes, since the comment was in the `AccessDenied` class.

Reply:
I think somethings up with github updating the lines I'm making comments in. I swear this one and the other one above was in the place I intended!